deploy-implicit-aurelia-app.cmd

```diff
@@ -0,0 +1,146 @@
+@if "%SCM_TRACE_LEVEL%" NEQ "4" @echo off
```

Comment:
Not sure it's really useful for a public sample.
If you're 100% convinced we should keep them, consider moving them under a "deployment" folder (in the Aurelia sample)